test/e2e/tests/add-custom-network.spec.js

```diff
@@ -17,6 +17,232 @@ describe('Custom network', function () {
       },
     ],
   };
+
+  it('should show warning when adding chainId 0x1', async function () {
+    await withFixtures(
+      {
+        dapp: true,
+        fixtures: new FixtureBuilder()
+          .withPermissionControllerConnectedToTestDapp()
+          .build(),
+        ganacheOptions,
+        title: this.test.title,
+      },
+      async ({ driver }) => {
+        await driver.navigate();
+        await driver.fill('#password', 'correct horse battery staple');
+        await driver.press('#password', driver.Key.ENTER);
+
+        await driver.openNewPage('http://127.0.0.1:8080/');
+        await driver.executeScript(`
+          var params = [{
+            chainId: "0x1",
+            chainName: "Fake Ethereum Network",
+            nativeCurrency: {
+              name: "",
+              symbol: "ETH",
+              decimals: 18
+            },
+            rpcUrls: ["https://customnetwork.com/api/customRPC"],
+            blockExplorerUrls: [ "http://localhost:8080/api/customRPC" ]
+          }]
+          window.ethereum.request({
+            method: 'wallet_addEthereumChain',
+            params
+          })
+        `);
+        const windowHandles = await driver.waitUntilXWindowHandles(
+          3,
+          1000,
+          10000,
+        );
```

Comment:
Nit: Do we need to double the timeout here? 5 seconds is the default; I would expect the notification to appear in that timeframe.
```suggestion
        const windowHandles = await driver.waitUntilXWindowHandles(3);
```